src/tests/topp/FileInfo_5_output.txt

```diff
@@ -13,15 +13,15 @@ Total number of peaks: 3149
 Number of spectra: 10
 
 Ranges:
-  retention time: 20.87 .. 39.97
+  retention time: 20.87 .. 39.97 sec (0.3 min)
```

Comment:
start also in min?

Reply:
maybe as 
```
(0.2 min .. 0.3 min)
```

Reply:
ah ok it's already the duration of the whole run

Reply:
should I use `(span 30 min)` to make it more clear?